0352-distributed-builds.md

```diff
@@ -0,0 +1,85 @@
+---
+title: ⚡️ L2 Distributed Builds & Caching
+---
+
+`cmake-re` supports the `--distributed --build` command line flags, scaling CMake builds to thousands of Jobs on RBE build clusters like the EngFlow platform.
+
+This supports fully distributed :
+- Compiling
+- Archiving
+- Linking
+
+## Remote Execution API (RE-API)
+The Remote Execution & Remote Caching is implemented with the _Remote Execution API_.
+
+The RE-API is an open and popular remote execution protocol to enable builds and test actions to be executed on remote servers instead of local machines, allowing distributed and parallel faster, more scalable builds.
```

Comment:
```suggestion
The RE API (Remote Execution API) is an open and popular remote execution protocol to enable build and test actions to be executed on remote servers instead of local machines, making builds faster and more scalable.
```